bin/setup

```diff
@@ -34,6 +34,7 @@ Dir.chdir APP_ROOT do
   run "cp keys/equifax_rsa.example keys/equifax_rsa"
   run "cp keys/equifax_rsa.pub.example keys/equifax_rsa.pub"
   run "cp certs/saml.crt.example certs/saml.crt"
+  run "which gpg || brew install gpg"
```

Comment:
The Homebrew installation will only work on OS X, but this part of the script runs on Docker as well. How about we move this down below, under the part where the script exits if it's being run on Docker?

Also, if the Docker image doesn't come with gpg, we should update the Docker setup to install gpg.